src/emulator/auth/widget_ui.ts

```diff
@@ -234,7 +234,7 @@ document.getElementById('main-form').addEventListener('submit', function(e) {
     if (screenName) claims.screenName = screenName;
     if (photoUrl) claims.photoUrl = photoUrl;
 
-    finishWithUser(createFakeClaims(claims));
+    finishWithUser(createFakeClaims(claims), email);
```

Comment:
```suggestion
    finishWithUser(createFakeClaims(claims), claims.email);
```

To make it default to `undefined` (omitted from JSON output) instead of `""`

Reply:
Sick, done!